v3/mechlovin/adelais/adelais_en_ciel-rev3.json

```diff
@@ -2,8 +2,6 @@
   "name": "Adelais En Ciel Rev.3",
   "vendorId": "0x4D4C",
   "productId": "0xAEC3",
-  "keycodes": ["qmk_lighting"],
```

Comment:
I recommend keeping this so the LIGHTING tab shows up in keycodes

![image](https://github.com/the-via/keyboards/assets/17491233/f8244d1b-8f77-4762-9ce8-82bf048cef10)